test/view.js

```diff
@@ -2,9 +2,9 @@
 
   var view;
 
-  module("Backbone.View", {
+  QUnit.module("Backbone.View", {
 
-    setup: function() {
+    beforeEach: function(assert) {
```

Comment:
No, but it does no harm leaving it there. It can be useful if `beforeEach`
has an async operation.
On Oct 7, 2015 12:25 AM, "Jordan Eldredge" <notifications@github.com> wrote:

> In test/view.js
> <https://github.com/jashkenas/backbone/pull/3816#discussion_r41352894>:
>
> >
> > -    setup: function() {
> > +    beforeEach: function(assert) {
>
> Do we need this assert callback if it's never used?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jashkenas/backbone/pull/3816/files#r41352894>.
>